app/src/app.js

```diff
@@ -124,11 +124,18 @@ export default class UnboxApp {
         // Solve CORS issues
         ctx.set('Access-Control-Allow-Origin', '*')
 
-        // Cache this please
-        ctx.set('Cache-Control', `max-age=${this.options['cache-control-age']}`)
```

Comment:
Removing this line means that non-index pages won't get cache control headers at all, which is surely not what you intended.